src/app/components/letters/create-letters-temp/create-letters-temp.component.html

```diff
@@ -1,10 +1,72 @@
 <app-top-bar></app-top-bar>
 <div class='container-crtLetter'>
-    <h1>Creating new letter's template</h1>
-    <div class='crtLetter__status'>
-      <strong>Name * </strong>
-      <mat-form-field appearance="outline">
-        <input matInput >
-      </mat-form-field>
-    </div>
+   <form class='crtLetter__form' [formGroup]='profileForm' (ngSubmit)='onSubmit()'>
```

Comment:
Please use double quotes in html, and single quotes in typescript files

Reply:
done